### PR TITLE
Sustain EU cookie message (Plan B)

### PR DESCRIPTION
**Pull request by JustinPinner**
We couldn't use headers from Fastly to identify EU-located browsers, so how about a server cookie instead? I'm sure there's some irony here.

Consider this alongside the corresponding Fastly PR.

@rich-nguyen What do you think of this approach?

### CODE REVIEW

**Comment by rich-nguyen** [comment]
:+1: , with those tweaks that we discussed, which allows us to do all this in one cookie.

**Comment by JustinPinner** [comment]
@zeftilldeath could you lend a hand to fix up the vertical alignment of the X in the close button (see image)? Thanks.

screen shot 2015-10-20 at 15 27 14

**Comment by JustinPinner** [comment]
Forgot to attach a pic of the close button after Zef's styling tweak, so here it is;

screen shot 2015-10-20 at 20 17 06

**Comment by paperboyo** [comment]
> Zef's styling tweak

Thanks, @JustinPinner, @zeftilldeath! ☯

File: static/src/javascripts/projects/common/modules/ui/cookiesBanner.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+     */
+    function init() {
+        var EU_COOKIE_MSG = 'GU_EU_MSG',
+            euMessageCookie = cookies.get(EU_COOKIE_MSG);
+
+        if (euMessageCookie && euMessageCookie === 'unseen') {
+            var link = 'https://www.theguardian.com/info/cookies',
+                txt = 'Welcome to the Guardian. This site uses cookies, read our policy <a href="' + link + '" class="cookie-message__link">here</a>',
+                opts = {important: true},
+                cookieLifeDays = 365,
```
Comment (OliverJAsh) [inline static/src/javascripts/projects/common/modules/ui/cookiesBanner.js]: Could we make this bigger? I don't fancy seeing the banner again once a year… :unamused:

**Pull request merged**